includes/Partials/woocommerce.js

```diff
@@ -50,13 +50,14 @@ function observeAndAdjustBacklink(event) {
 }
 
 (function () {
+  let brand_name = brand == 'crazy-domains' ? 'crazy-domains' : 'bluehost'
```

Comment:
We can use `container->plugin()->id` instead of brand to avoid dealing with naming issues